File: app/src/main/java/com/jobtick/fragments/ServiceFeeInfoBottomSheet.java
package com.jobtick.fragments;


import android.os.Bundle;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import androidx.annotation.Nullable;
import androidx.fragment.app.DialogFragment;

import com.google.android.material.bottomsheet.BottomSheetDialogFragment;
import com.jobtick.R;

public class ServiceFeeInfoBottomSheet extends BottomSheetDialogFragment {


    public ServiceFeeInfoBottomSheet() {

    }


    @Override
    public void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setStyle(DialogFragment.STYLE_NORMAL, R.style.ThemeOverlay_BottomSheetDialog);
    }

    @Override
    public View onCreateView(LayoutInflater inflater, ViewGroup container,
                             Bundle savedInstanceState) {

        return inflater.inflate(R.layout.service_fee_info_bottom_sheet, container, false);
    }

}

File: app/src/main/java/com/jobtick/android/fragments/ServiceFeeInfoBottomSheet.kt
package com.jobtick.android.fragments

import com.google.android.material.bottomsheet.BottomSheetDialogFragment
import android.os.Bundle
import com.jobtick.android.R
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup

class ServiceFeeInfoBottomSheet : BottomSheetDialogFragment() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setStyle(STYLE_NORMAL, R.style.ThemeOverlay_BottomSheetDialog)
    }

    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?,
                              savedInstanceState: Bundle?): View? {
        return inflater.inflate(R.layout.service_fee_info_bottom_sheet, container, false)
    }
}